cluster-operator/.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
Do we really need this when we have `**/target/*` in the top level `.gitignore`?